subprojects/core/build.gradle.kts

```diff
@@ -131,6 +131,7 @@ dependencies {
     implementation(projects.inputTracking)
     implementation(projects.modelGroovy)
     implementation(projects.serviceRegistryBuilder)
+    implementation(projects.problemsRendering)
```

Comment:
same as above

Reply:
Answered above, follow up in the main thread